### ISSUE TITLE
Space used to preserve files option improvements

### ISSUE THREAD

**Issue by gmatagmis** [unetbootin/unetbootin#88]
Unebootin for Windows, version 652, Russian language, Windows XP.
Maximum value for "Space used to preserve files option improvement afer reboots (Ubuntu only)" in GUI text field is limited by 4 characters (maximum "9999"). It would be nice to be able to set more, when you are using flash drive for 16+ Gb. 
Whole description "Space used to preserve files option improvement afer reboots (Ubuntu only)" on Russian does not fit in one row, and half of text is hidden with default program window size.
It also would be nice to have checkbox "Use all space available", and show whole option GUI elements only when Ubuntu distributive is choosen.

**Comment by froger-me** [comment]
ˆ+1

**Comment by wpalmer** [comment]
At the very least, there should be a "use all available space" option

**Comment by qwertychouskie** [comment]
The issue here is that FAT32 formated-flash drives don't support more than 4096 MB.

**Comment by csylvain** [comment]
why did it accept a value of 9728 MB then?  no mention of 4096 MB in the tooltip!
therefore, "use all available space" should display 4096 in the text entry box if there is at least that much available.

**Comment by Dansiman** [comment]
I am pretty sure that I entered 9999 into that box and got a 9.999GB persistence file...

**Comment by csylvain** [comment]
not a valid FAT32 file, then. or at least (9999 minus 4096) MB unusable space. MAXINT is the limit.

**Comment by Dansiman** [comment]
Since UNetbootin currently only supports FAT-formatted USB drives, then, this field should be changed from a simple 4-digit limit to an actual value limit of 4096.

**Comment by csylvain** [comment]
yes, this. oh and technically UINT32_MAX is the limit for FAT32.
